File: c/Order/1simpleOrder.c
#include <stdio.h>

int main(){
    int num1, num2;

    printf("Digite um número: ");
    scanf("%d", &num1);
    printf("Digite outro número: \n");
    scanf("%d", &num2);

    if(num1 > num2){
        printf("%d, %d", num1, num2);
    } else{
        printf("%d, %d", num2, num1);
    }

    return 0;
}

File: python/Order/1simpleOrder.py
'''

Faça um programa que leia 10 números e mostre esses números em ordem crescente e em ordem decrescente.

a) se quiser, faça uma versão com 2 números apenas, usando variáveis simples
b) se quiser, faça uma versão com 3 números apenas, usando variáveis simples
c) para quantidades maiores de números, é recomendado utilizar uma estrutura de dados como um vetor para armazenar os números


'''

num1 = int(input("Digite um número: "))
num2 = int(input("Digite outro número: "))

if (num1 > num2):
    print(num1, num2)
else:
    print(num2, num1)
